app/views/layouts/application.html.erb

```diff
@@ -68,6 +68,7 @@
     </div>
 
     <%= external_javascript_include_tag "jquery", "2.1.1" %>
+    <%= content_for :js_widget_patch %>
```

Comment:
I agree about the hackiness. My personal best-case scenario is that we fix the issue by replacing SlickGrid. It would remove our dependence on jQuery UI, and also free us from potential problems down the road (since support has been dropped). Until then, though, it appears that this solution works.

Just about every page loads Bootstrap (maybe not every page needs the JS), but certainly only some pages need jQuery.